framework/src/st_things/things_stack/things_types.h

```diff
@@ -19,6 +19,7 @@
 #ifndef _THINGS_TYPES_H_
 #define _THINGS_TYPES_H_
 
+#include <wifi_manager/wifi_manager.h>
```

Comment:
If you try to reference define in wifi_manager.h in our code, clean up everything.
remove line 41 ~ 46
change other defines

Reply:
fixed.